module1-introduction-to-backend/r1.1-beginners-guide-nodejs-npm/README.md

```diff
@@ -0,0 +1,110 @@
+- an online repository that hosts JavaScript packages</br>
+
+A package is nothing but a directory that contains a bunch of modules. Some popular npm packages are [lodash](https://lodash.com/) and [moment](https://momentjs.com/). Node.js has a wide community that develop good packages for everybody to use.
+
+When we have a remote package in our project, it is called as a dependency since our project depends on it. We need to keep track of our dependencies or at least list them down somewhere. We list all our dependencies inside a `package.json` file which is a JSON file that contains some information about our project and dependencies it needs. This file is essential for NPM.
+
+Node.js also ships with a collection of [built-in packages](https://nodejs.org/api/index.html) called as a Node Standard Library. These packages are essential to perform low-level operations like File System I/O and Networking. We do not have to install them using NPM.
+
+### How much JavaScript should I know to use Node.js?
+As a beginner, it can be difficult to understand where does JavaScript end, and where Node.js begins, and vice versa. However, when you're writing code for a Node.js application you are writing JavaScript. So, it is important to grasp the fundamentals of JavaScript syntax such as:
+- Lexical Structure
+- Expressions
+- Types
+- Classes
+- Variables
+- Functions
+- this
+- Arrow Functions
+- Loops
+- Scopes
+- Arrays
+- Objects
+- Array and Object methods
+- Template Literals
+- Semicolons
+- Strict Mode
+- ECMAScript 6 or ES6
+
+Also, as asynchronous programming is the core of Node.js it is important to understand concepts like:
+- [Asynchronous programming and callbacks](https://nodejs.dev/learn/javascript-asynchronous-programming-and-callbacks)
+- [Timers](https://nodejs.dev/learn/discover-javascript-timers)
+- [Promises](https://nodejs.dev/learn/understanding-javascript-promises)
+- [Async and Await](https://nodejs.dev/learn/modern-asynchronous-javascript-with-async-and-await)
+- [Closures](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures)
+- [The Event Loop](https://nodejs.dev/learn/the-nodejs-event-loop)
+
+If you're interested to dive deeper into understanding Node.js, [here](https://nodejs.dev/learn/introduction-to-nodejs) is a learning path that you can follow.
+
+## Why are we teaching Node.js in this bootcamp?
+From the previous lesson or previous experience, you might already be familar with other server-side languages like Python, PHP, Ruby or Java. So why did we choose to teach Node.js?
+
+1. Node.js has been consistently growing in popularity since it's introduction. As per the [Stackoverflow developer survey of 2021](https://insights.stackoverflow.com/survey/2021), JavaScript is the most popular language for 9 years in a row and Node.js has moved up to be the 6th most popular technology. Many companies that we have got in touch with have expressed interest in hiring more Node.js developers.
```

Comment:
IMO any time more than 2 or 3 bullets are used (even if only 2 or 3) you should include a "summary term" at the beginning of each of them, basically exactly as you did on line 42, otherwise its just a huge blob of text that happens to be bullet pointed